packages/frontend/src/components/wallet-migration/WalletMigration.jsx

```diff
@@ -39,6 +37,20 @@ const encodeAccountsToURL = async (accounts, publicKey) => {
         ]);
     }
 
+    return accountsData;
+};
+
+const encodeAccountsToSender = async (accounts, publicKey) => {
```

Comment:
encodeAccountsToURL seems like it needs to add an additional argument such as wallet or walletID this way we can do a lookup for url and keep the codebase DRY

Reply:
Shouldn't block this PR based on this. Will address in a separate PR.

Reply:
Agree with @agileurbanite here. Would appreciate if you can add a `getUrl` key to [`WALLET_OPTIONS`](https://github.com/near/near-wallet/pull/2883/files#diff-133f50f707eb791d6963507c2cdb6f305d087105bb4484ffc3a44a1af9928d55R37) and then call that from a single `encodeAccountsToURL` please

Reply:
@esaminu i thought about this some more and we shouldn't block this PR, the original implementation made the function very MNW specific.  I'd recommend we handle the refactor.